tests/json_fmt.cpp

```diff
@@ -0,0 +1,38 @@
+
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
+#include "fast_float/fast_float.h"
+
+int main()
+{
+  const std::vector<double> expected{ -0.2, 0.02, 0.002, 1., 0. };
```

Comment:
Both 1. and 0. are fine, but my expectation is that you could use 1 and 0. (This is not a request for you to change the code, just a comment.)